chef/cookbooks/crowbar/templates/default/crowbar-evacuate-host.sh.erb

```diff
@@ -45,6 +45,19 @@ ORIGINAL_INSTANCES=$UPGRADEDIR/original_instances
 # List active instances on host before the live-migration starts
 openstack server list --insecure --all-projects --status active --host $host -f value -c ID > $ORIGINAL_INSTANCES
 
+i=0
+while [[ $i -lt 10 ]] && openstack --insecure compute service list --service nova-compute -f value -c State | grep down ; do
```

Comment:
You could write this as `(( $i < 10 ))` which makes it look a lot less like Fortran 77 :wink: but that's really just a minor stylistic thing :smile:.